src/main/java/com/indiraactive/fulfillmentplatform/dao/job/JobJpa.java

```diff
@@ -0,0 +1,140 @@
+
+    @Column(name = "active")
+    private boolean active;
+
+    public JobJpa() { }
+
+    public JobJpa(Integer id, Supplier supplier, String createdByUser, Date startDateTime, boolean runOnce, String cronExpression, boolean active) {
+        this.id = id;
+        this.supplier = supplier;
+        this.createdByUser = createdByUser;
+        this.startDateTime = startDateTime;
+        this.runOnce = runOnce;
+        this.cronExpression = cronExpression;
+        this.active = active;
+    }
+
+    public JobJpa(Supplier supplier, String createdByUser, Date startDateTime, boolean runOnce, String cronExpression, boolean active) {
+        this.supplier = supplier;
+        this.createdByUser = createdByUser;
+        this.startDateTime = startDateTime;
+        this.runOnce = runOnce;
+        this.cronExpression = cronExpression;
+        this.active = active;
+    }
+
+    public Integer getId() {
+        return id;
+    }
+
+    public void setId(Integer id) {
+        this.id = id;
+    }
+
+    public Supplier getSupplier() {
+        return supplier;
+    }
+
+    public void setSupplier(Supplier supplier) {
+        this.supplier = supplier;
+    }
+
+    public String getCreatedByUser() {
+        return createdByUser;
+    }
+
+    public void setCreatedByUser(String createdByUser) {
+        this.createdByUser = createdByUser;
+    }
+
+    public Date getStartDateTime() {
+        return startDateTime;
+    }
+
+    public void setStartDateTime(Date startDateTime) {
+        this.startDateTime = startDateTime;
+    }
+
+    public boolean isRunOnce() {
+        return runOnce;
+    }
+
+    public void setRunOnce(boolean runOnce) {
+        this.runOnce = runOnce;
+    }
+
+    public String getCronExpression() {
+        return cronExpression;
+    }
+
+    public void setCronExpression(String cronExpression) {
+        this.cronExpression = cronExpression;
+    }
+
+    public boolean isActive() {
+        return active;
+    }
+
+    public void setActive(boolean active) {
+        this.active = active;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        JobJpa jobJpa = (JobJpa) o;
+
+        if (runOnce != jobJpa.runOnce) return false;
+        if (id != null ? !id.equals(jobJpa.id) : jobJpa.id != null) return false;
+        if (supplier != null ? !supplier.equals(jobJpa.supplier) : jobJpa.supplier != null) return false;
+        if (createdByUser != null ? !createdByUser.equals(jobJpa.createdByUser) : jobJpa.createdByUser != null)
+            return false;
+        if (startDateTime != null ? !startDateTime.equals(jobJpa.startDateTime) : jobJpa.startDateTime != null)
+            return false;
+        return cronExpression != null ? cronExpression.equals(jobJpa.cronExpression) : jobJpa.cronExpression == null;
+    }
+
+    @Override
+    public int hashCode() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method hashCode() has an NPath complexity of 15625](https://app.codacy.com/app/indira-active/fulfillment-platform/pullRequest?prid=1507512)